theme_notes/views/snippets/s_product_catalog.xml

```diff
@@ -4,12 +4,12 @@
 <template id="s_product_catalog" inherit_id="website.s_product_catalog">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_cc o_cc5 pb160 pt160 parallax s_parallax_is_fixed s_parallax_no_overflow_hidden o_full_screen_height" remove="pb32 pt48" separator=" "/>
+        <attribute name="class" add="o_cc o_cc5 pb160 pt160 parallax s_parallax_is_fixed s_parallax_no_overflow_hidden o_full_screen_height" remove="pb64 pt64" separator=" "/>
         <attribute name="data-oe-shape-data">{"shape":"web_editor/Wavy/10","flip":[]}</attribute>
         <attribute name="data-scroll-background-ratio">1</attribute>
     </xpath>
     <!-- Shape & Background image -->
-    <xpath expr="//section/*" position="before">
+    <xpath expr="//div[hasclass('o_we_bg_filter')]" position="replace">
```

Comment:
Did you notice that it's not readable. Especially in mobile view
![image](https://github.com/user-attachments/assets/c62fa57f-4d59-4bca-a93b-fce6f2fb08f8)

Reply:
Good catch,  I added a color filter 👍